charts/tractusx-connector/values.yaml

```diff
@@ -286,12 +286,13 @@ controlplane:
   url:
     # -- Explicitly declared url for reaching the ids api (e.g. if ingresses not used)
     ids: ""
+    readiness: ""
```

Comment:
we currently use it for the helm tests. I'm not very happy with it, because the observability endpoints should not be exposed. The `helpers.tpl` template provides that shorthand by constructing the url to the control plane and data plane readiness endpoint.

The alternative would be to directly construct the URLs in the `template/tests/test-*-readiness.yaml` pods:
```yaml
#  instead of 
args: ['{{ include "tdxc.controlplane.url.readiness" . }}']

# we would write
{{- printf "http://%s-controlplane:%v%s/check/readiness" (include "txdc.fullname" $ ) $.Values.controlplane.endpoints.observability.port $.Values.controlplane.endpoints.observability.path -}}
```